README.md

```diff
@@ -1,18 +1,53 @@
-<h1 align="center">Invidious</h1>
+<div align="center">
+  <img src="assets/android-chrome-192x192.png">
+  <h1>Invidious</h1>
+
+  <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">
+    <img alt="License: AGPLv3+" src="https://shields.io/badge/License-AGPL%20v3+-blue.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/actions">
+    <img alt="Build Status" src="https://github.com/iv-org/invidious/workflows/Invidious%20CI/badge.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/issues">
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/iv-org/invidious?color=important">
+  </a>
+  <a href="https://github.com/iv-org/invidious/issues">
+    <img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/iv-org/invidious?color=blueviolet">
+  </a>
+  <a href="https://hosted.weblate.org/engage/invidious/">
+    <img alt="Translation Status" src="https://hosted.weblate.org/widgets/invidious/-/translations/svg-badge.svg">
+  </a>
+
+  <h3>An open source alternative front-end to YouTube</h3>
+
+  <a href="https://instances.invidious.io/">Instances list</a>
+  &nbsp;•&nbsp; 
+  <a href="#documentation">Documentation</a>
+  &nbsp;•&nbsp;
+  <a href="#contribute">Contribute</a>
+  &nbsp;•&nbsp;
+  <a href="#donate">Donate</a>
+
+  <h5>Chat with us:</h5>
+  <a href="https://matrix.to/#/#invidious:matrix.org">
+    <img alt="Matrix" src="https://img.shields.io/matrix/invidious:matrix.org?label=Matrix">
+  </a>
+  <a href="https://web.libera.chat/?channel=#invidious">
+    <img alt="Libera.chat (IRC)" src="https://img.shields.io/badge/IRC%20%28Libera.chat%29-%23invidious-green">
+  </a>
+</div>
+
+
+## Screenshots
+
+| Player                              | Preferences                         | Subscriptions                         |
+|-------------------------------------|-------------------------------------|---------------------------------------|
+| ![](screenshots/01_player.png)      | ![](screenshots/02_preferences.png) | ![](screenshots/03_subscriptions.png) |
+| ![](screenshots/04_description.png) | ![](screenshots/05_preferences.png) | ![](screenshots/06_subscriptions.png) |
+
+
+## Features
 
-<h2 align="center">Invidious is an alternative front-end to YouTube.</h2>
-
----
-
-## Invidious instances:
-
-Public Invidious instances are listed on the documentation website: https://instances.invidious.io/
-
----
-
-## Invidious features:
-
-- [Copylefted libre software](https://github.com/iv-org/invidious) (AGPLv3+ licensed)
 - Lightweight (the homepage is ~4 KB compressed)
```

Comment:
It was 4 KB years ago... without the pictures, let's just remove " (the homepage is ~4 KB compressed)" and just keep "Lightweight"